cmd/create/cluster/cmd.go

```diff
@@ -610,6 +674,9 @@ func run(cmd *cobra.Command, _ []string) {
 		Expiration:         expiration,
 		ComputeMachineType: computeMachineType,
 		ComputeNodes:       computeNodes,
+		Autoscaling:        autoscaling,
+		MinReplicas:        minReplicas,
```

Comment:
min/max replicas value will be set to 0 it wasn't provided. will the backend support it?

Reply:
0 is illegal in the backend and considered "NOT SET"

Reply:
The value should be validated before getting here. Also keep in mind that if the user does something like `rosa create cluaster --min-replicas=0`, they didn't set `--enable-autoscaling`, so these values here may be misleading (see above comment regarding forcing interactive mode on the user)

Reply:
Added a validation